src/components/EditGrower.js

```diff
@@ -38,25 +38,40 @@ const EditGrower = (props) => {
   const [growerUpdate, setGrowerUpdate] = useState(null);
   const [loadingGrowerImages, setLoadingGrowerImages] = useState(false);
   const [saveInProgress, setSaveInProgress] = useState(false);
+  const [customUrl, setCustomUrl] = useState('nothing');
 
   useEffect(() => {
     async function loadGrowerImages() {
       if (grower?.id) {
         setLoadingGrowerImages(true);
         const selfies = await growerContext.getGrowerSelfies(grower.id);
         setLoadingGrowerImages(false);
-
+        const notAllowNull =
+          selfies !== undefined && selfies.includes(null)
+            ? selfies.filter((e) => e !== null)
+            : selfies;
         setGrowerImages([
-          ...(grower.imageUrl ? [grower.imageUrl] : []),
-          ...(selfies || []).filter((img) => img !== grower.imageUrl),
+          ...(grower.image_url ? [grower.image_url] : []),
+          ...(notAllowNull || []).filter((img) => img != grower.image_url),
+          customUrl === '' ? null : customUrl,
         ]);
       }
     }
 
     setGrowerUpdate(null);
     loadGrowerImages();
-  }, [grower]);
-
+  }, [grower, customUrl]);
+  function isImgURL(url) {
+    if (typeof url === 'object') {
+      return url === null
+        ? false
+        : /\.(gif|jpe?g|tiff?|png|webp|bmp)$/i.test(url.image_url);
+    }
+    if (url === null || url === undefined) {
```

Comment:
I change that function, i delete that if(), because typeof null is an object too. So i was repeating code for no reason
![image](https://user-images.githubusercontent.com/100769556/222024238-2c7d9012-5f44-4949-bab1-f5517255061b.png)